src/ory-editor/convertAction.js

```diff
@@ -0,0 +1,39 @@
+import { renderServersideContent } from "./index";
+import $ from "jquery";
+import EntityEditor from "./index";
+
+export default (id, href, callback) => {
+  $.ajax({
+    url: href,
+    type: 'GET',
+    async: true,
+    beforeSend: () => {
+      $('#loading').show()
+    },
+    complete: () => {
+      $('#loading').hide()
+    }
+  }).done(function (data) {
+    function getEditedArticle ($all) {
+      const $editable = $all.find(
+        `.editable[data-id=${id}][data-edit-type="ory"]`
+      )
+      return $editable.closest('article').length
+        ? $editable.closest('article')
+        : $all.find('#content-layout article')
+    }
+
+    const $target = getEditedArticle($('body'))
+    const $dataArticle = getEditedArticle($(data))
+    $target.html($dataArticle.html())
+    // TODO: Can we really drop this?
+    // renderServersideContent()
+
+
+    // FIXME: this is definitely not defined
+    const $editButton = $(`.ory-edit-button[data-id=${id}]`)
```

Comment:
@Knorrke: when converting, this button is undefined. Therefore, `editPath` is also undefined and form pulling fails. Any ideas why this happens or how we can get the edit path otherwise?

Reply:
Ah ok, this explains, why it works on the `entity/view/:id/convert` route itself but not when using the request. I'll take a look. We get the correct edit-button in the convert response, I thought I replaced the Buttons somewhere... but we could get it from the response directly with the same query (just on data)...